.github/workflows/rust.yml

```diff
@@ -139,7 +139,6 @@ jobs:
         run: |
           forest-cli sync wait
           forest-cli --chain calibnet db stats
-          bash scripts/health_check.sh localhost
```

Comment:
Let's delete this file then as it is not used anywhere anymore in this repository.

Reply:
Done.